csuntechlab/iSTART
# STR-177 -- FE: Fix quiz indicator should only pop up for answer selected

# Description
Fixed quiz indicator; Fixed continuous quiz slides bug
  
# Testing
1. Login as `cyndee.bellamy` or `april.feldman`.
2. Visit dashboard and click on the ***Alcohol Module***.
3. Visit `slide 6` of the content the client provided ***index 4*** currently on this branch.
4. Ensure when you click on a quiz option, only the indicator appears on the one you clicked.
5. Ensure all previous functionality works; **(clicking on an option allows you to continue)**.
6. Make sure on `slide 6`,  ***index 4***, the indicator applies to the correct answer.
